src/main/scala/cognite/spark/v1/AssetsRelation.scala

```diff
@@ -77,7 +78,8 @@ class AssetsRelation(config: RelationConfig)(val sqlContext: SQLContext)
       name = m.get("name"),
       source = m.get("source"),
       dataSetIds = m.get("dataSetId").map(idsFromWrappedArray(_).map(CogniteInternalId(_))),
-      labels = m.get("labels").map(externalIdsToContainsAny).getOrElse(None)
+      labels = m.get("labels").map(externalIdsToContainsAny).getOrElse(None),
```

Comment:
do we need `getOrElse(None)` here? I know it was already like that.

Reply:
it's the same as flatten in this case - it unwraps one layer of options

Reply:
Actually, I don't know why didn't we use flatMap, i'll change it

Reply:
hehe probably it was me not using scala functions effectively at the time.